build.gradle

```diff
@@ -36,7 +36,9 @@ allprojects {
       compile 'com.googlecode.json-simple:json-simple:1.1.1'
       compile 'org.xerial.snappy:snappy-java:1.1.2.6'
       compile 'org.yaml:snakeyaml:1.18'
-      compile 'org.apache.cassandra:cassandra-all:2.1.17'
+      compile ('org.apache.cassandra:cassandra-all:2.1.17'){
```

Comment:
I could instead exclude this from any dependency

```
    configurations {
        all*.exclude group: 'ch.qos.logback', module: 'logback-classic'
    }
```

Reply:
For reference, internally we always exclude the module for configurations.all: 
    exclude module: 'logback-classic' //by artifact name
    exclude module: 'logback-core' //by artifact name

The reason why we never experienced this. 
@zmarois your thoughts?

Reply:
I'm not quite following your suggestion. Do you suggest

1. Along my comment above, just add the exclude to configurations.all
2. Also exclude 'logback-core'
3. Just do the exclude in my internals and leave this repo without the exclude

I'm totally on board with 1 and/or 2. I have done 3 already, just considering there was an issue explicitly raised against this, seemed worthwhile to share it.

Reply:
sorry, i was not clear. I am suggesting (1) and (2). I believe this would be a problem for others as well so better to push it in the trunk so we don't duplicate efforts.

Reply:
Gotcha. I'll push that change. Thanks for the review!

Reply:
Alright, I've excluded both logback-classic and logback-core. I don't have a ton of experience with gradle, so my syntax might not be as terse as it could be, but I confirmed the jars are excluded from the final build artifact.